tcms/templates/plan/preview.html

```diff
@@ -48,5 +49,5 @@
 <hr />
 {% endif %}
 {% empty %}
-{"msg":"The plan you specified does not exist in the database."}
+{"msg":"{% trans "The plan you specified does not exist in the database." %}"}
```

Comment:
I don't think this is a valid construct in Django templates. Just looks like printing JSON inside HTML.